app/presenters/concerns/claim_csv_helpers.rb

```diff
@@ -0,0 +1,33 @@
+module ClaimCSVHelpers
```

Comment:
whats the reason for breaking out this concern. you generally don't use a concern/mixin unless it needs its logic shared with another class, or, perhaps if its super long  - see [this article ](https://rollout.io/blog/when-to-be-concerned-about-concerns/). This does not seem that long and the abstraction is does not aid readbility IMO

Reply:
Code climate was complaining, it has a max 24 methods per class, I added 2 which took it to 26.

Reply:
understandable but the stuff you have removed seems to be the core functionality of this class - i would either amend rubocop.yml to exclude this file from that cop and not use a concern at all, or create a concern called Something like `ClaimCsvColumns` and put all the "column" methods in there, and include that in here

Reply:
👍 will create a ClaimCsvColumns concern

Reply:
Instead as suggested I have added a concern management_information_reportable (to follow 'able' naming convention) containing all the column methods. Code climate has also made me refactor the af1_lf1_processed_by method (not sure why, as I haven't changed it) as it was complaining about cognitive complexity.